.ebextensions/07_app_logs.config

```diff
@@ -1,21 +0,0 @@
-# Add our app logs to Elastic Beanstalk's list of logfiles to tail, bundle and rotate.
-
-files: 
-  "/opt/elasticbeanstalk/tasks/bundlelogs.d/applogs.conf" :
-    mode: "000644"
-    owner: root
-    group: root
-    content: |
-      /var/app/current/logs/*.log
-  "/opt/elasticbeanstalk/tasks/publishlogs.d/applogs.conf" :
-    mode: "000644"
-    owner: root
-    group: root
-    content: |
-      /var/app/current/logs/*.log
-  "/opt/elasticbeanstalk/tasks/taillogs.d/applogs.conf" :
-    mode: "000644"
-    owner: root
-    group: root
-    content: |
-      /var/app/current/logs/*.log
```

Comment:
This configuration is removed, since we're now piping logs to stdout rather than intermediate files in the app directory.